libvast/src/system/application.cpp

```diff
@@ -124,7 +124,10 @@ auto make_export_command() {
   export_->add_subcommand("null",
                           "exports query without printing them (debug option)",
                           opts("?vast.export.null"));
-  export_->add_subcommand("arrow", "exports query results in Arrow format",
+  export_->add_subcommand("arrow",
+                          "exports query results in Arrow format with separate "
+                          "IPC streams for each schema, all concatenated "
+                          "together",
```

Comment:
What does "all concatenated together" add in terms of understanding? Isn't that already implied by "IPC streams"?

Reply:
It's just a single output stream instead of one output stream per schema as you'd usually expect. I found this pretty clear and an improvement over the current helptext, but maybe you have an idea to improve it further.

Reply:
I'm fine with the wording, just want to adjust my mental model to better understand IPC streams.